apps/consumer-client/pages/zuzalu-examples/signature-proof.tsx

```diff
@@ -16,8 +16,9 @@ import { PASSPORT_SERVER_URL, PASSPORT_URL } from "../../src/constants";
  * request a Semaphore Signature PCD as a third party developer.
  */
 export default function Page() {
+  const popupSrcId = "zuzalu-signature-proof";
```

Comment:
`popupSrcId` here vs `srcId` elsewhere - a little confusing

Reply:
`srcId` in the context of a PCD Request makes sense. `srcId` within a component doesn't give much context on what it is?